Fix inconsistent behaviour between local and remote scraper searches (#8316)
sunpy/sunpy: changelog/8316.bugfix.rst

> Fix inconsistent behaviour between local and remote (http/ftp) ~`sunpy.net.scraper.Scraper` searches. Local searches will no longer crash is directory is not found during search.

Author: ```suggestion
Fix inconsistent behaviour between local and remote (http/ftp) ~`sunpy.net.scraper.Scraper` searches. Local searches will no longer crash if an expected directory does not exist during search.
```
